### ISSUE TITLE
Malformed HTML can make the forum unusable

### ISSUE THREAD

**Issue by emanuele45** [BurkeKnight-Enterprises/BKE-Elk-Global_Header_and_Footer#1]
http://www.elkarte.net/community/index.php?topic=4042.0

I can think of two alternatives:
1. split `$modSettings['global_head'];` into a CSS section and a js section and strip tags from the css and wrap it into a style tag, then use addInlineJavascript on the js one, or
2. consider to use http://htmlpurifier.org/ in order to cleanup the code before **at least** sending it to the head.

In favour of 1 I'd say that Elk 1.1 will have also an equivalent addCSSRules in order to deal with CSS.

**Comment by BurkeKnight** [comment]
Will look into this when get back to regular PC.

**Comment by artixnous** [comment]
With Elk 1.1 this renders the forum unusable (no forum page is loaded, no error, just a blank screen). That's even with a completely empty header section.

**Comment by emanuele45** [comment]
Hi @archnous the one you describe seems a completely different issue, the one above just means if you have inserted in the header mismatched html tags you may end up not being able to *see* the forum.
By your description it seems that you don't see anything just by installing the addon?

**Comment by artixnous** [comment]
Yes, in 1.1 only; in 1.0.10 it worked fine and I used it for inserting g-analytics snippet, as your addon wasn't working. Spud's update of g-a to 0.0.4 fixed it.

**Comment by artixnous** [comment]
Sorry for necro-bumping, but is there any chance of updating this for Elkarte 1.1?
